docs/integrations/web-servers/apache-tomcat.md

```diff
@@ -274,27 +256,8 @@ This section explains the steps to collect Apache Tomcat logs from a Kubernetes
 1. Sumo Logic Kubernetes collection will automatically start collecting logs from the pods having the annotations defined above.
 1. Verify logs in Sumo Logic.
 
-**Add an FER to normalize the fields in Kubernetes environments**. Labels created in Kubernetes environments automatically are prefixed with pod_labels. To normalize these for our app to work, we need to create a Field Extraction Rule if not already created for WebServer Application Components. To do so:
-
-1. [**Classic UI**](/docs/get-started/sumo-logic-ui-classic). In the main Sumo Logic menu, select **Manage Data > Logs > Field Extraction Rules**. <br/>[**New UI**](/docs/get-started/sumo-logic-ui). In the top menu select **Configuration**, and then under **Logs** select **Field Extraction Rules**. You can also click the **Go To...** menu at the top of the screen and select **Field Extraction Rules**.  
-2. Click the + Add button on the top right of the table.
-3. The **Add Field Extraction Rule** form will appear:
-4. Enter the following options:
-  * **Rule Name**. Enter the name as **App Observability - Webserver**.
-  * **Applied At.** Choose **Ingest Time**
-  * **Scope**. Select **Specific Data**
-  * **Scope**: Enter the following keyword search expression:
-    ```sql
-    pod_labels_environment=* pod_labels_component=webserver pod_labels_webserver_farm=* pod_labels_webserver_system=*
-    ```
-  * **Parse Expression**.Enter the following parse expression:
-    ```sql
-    if (!isEmpty(pod_labels_environment), pod_labels_environment, "") as environment
-    | pod_labels_component as component
-    | pod_labels_webserver_system as webserver_system
-        | pod_labels_webserver_farm as webserver_farm
-    ```
-5. Click **Save** to create the rule.
+<br />**FER to normalize the fields in Kubernetes environments**. Labels created in Kubernetes environments automatically are prefixed with `pod_labels`. To normalize these for our app to work, we will have a Field Extraction Rule automatically created for Apache Tomcat Web Server Application Components named as **AppObservabilityApacheTomcatWebserverFER**
+<br />
```

Comment:
These have to be closed. Can you please fix the other instances? There are about 9 or 10 if you scroll down.

```suggestion
<br/>**FER to normalize the fields in Kubernetes environments**. Labels created in Kubernetes environments automatically are prefixed with `pod_labels`. To normalize these for our app to work, we will have a Field Extraction Rule automatically created for Apache Tomcat Web Server Application Components named as **AppObservabilityApacheTomcatWebserverFER**
<br/>
```

Reply:
@sachin-sumologic